embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/storage/StorageService.kt

```diff
@@ -0,0 +1,27 @@
+package io.embrace.android.embracesdk.storage
+
+import java.io.File
+import java.io.FilenameFilter
+
+/**
+ * Provides File instances for files in the cache and files directories.
+ * Previous versions of the SDK used to store files in the cache directory, but now we use the
+ * files directory for everything except the config cache.
+ */
+internal interface StorageService {
+
+    val cacheDirectory: File
```

Comment:
Is it possible to remove this from interface and just have it as part of the implementation?

Reply:
Updated.